etl/src/main/java/org/pih/hivmigration/etl/sql/SqlMigrator.java

```diff
@@ -81,10 +82,17 @@ Connection getConnection(Properties p) throws SQLException {
     }
 
     void executeMysql(String update) throws SQLException {
+        executeMysql(update, true);
+    }
+
+    void executeMysql(String update, boolean logStatements) throws SQLException {
         QueryRunner qr = new QueryRunner();
         try (Connection connection = getConnection(getMysqlConnectionProperties())) {
             List<String> stmts = SqlStatementParser.parseSqlIntoStatements(update, ";");
             for (String sqlStatement : stmts) {
+                if (logStatements) {
```

Comment:
I think this is best handled by standard log4j configuration.  If you want to suppress or enable logging, we should use log levels.  Make this into a log.debug or a log.trace call, and make the default logging level into info in order to accomplish this.  I think we should remove this "logStatements" boolean and just keep the single method.

Reply:
So the idea here is that calls to `executeMysql(String name, String update)` should log the `name`, and calls to `executeMysql(String update)` should log the first chunk of each MySQL command that constitutes the update.

Reply:
Ha, @brandones  when I added executeMysql(name, update) I was at first thinking of doing just that!

This makes sense to me... by default if you call executeMysql(update) it logs the first chunk of MySQL, but if you call executeMysql(name, update) it logs the name instead.

A few ways that we could probably refactor this to make it a little cleaner, but probably not worth it now...

Reply:
Fine, not really a big deal.